charts/registry-scanner/templates/_helpers.tpl

```diff
@@ -35,6 +35,7 @@ Common labels
 */}}
 {{- define "registry-scanner.labels" -}}
 helm.sh/chart: {{ include "registry-scanner.chart" . }}
+product-type: registry-scanner-orchestrator
```

Comment:
Why do we need this label? Should we consider to use one of the well-known label for that? https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels

Reply:
purpose is, we want to differentiate from orchestrator vs. worker job logs by label.

i like the conventions from k8s @mavimo , thx!
what about using
```
    app.kubernetes.io/name: registry-scanner
    app.kubernetes.io/component: orchestrator | worker
```

would that suit better?